README.md

```diff
@@ -1,13 +1,13 @@
 # Project Name
 
-> Project description
+######An online retail front end, with a new, production worthy backend.
```

Comment:
I like how you've made a distinction between front end and backend. I'm going to incorporate something similar into mine. Thanks!